### PR TITLE
Ruowan/arm endpoint

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log - oav
 
+## 5/13/2021 2.5.1
+
+- oav support overwrite location, armEndpoint, subscriptionId
```
Comment (raych1) [inline ChangeLog.md:5]: Can you remove `oav`?

File: lib/testScenario/postmanCollectionRunnerClient.ts [hunk 2]
```diff
@@ -478,27 +480,27 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
...
       )
       .on("done", async (_err, _summary) => {
-        if (this.opts.enableBlobUploader) {
-          const keys = await this.swaggerAnalyzer.getAllSecretKey();
-          const values: string[] = [];
-          for (const [k, v] of Object.entries(this.collectionEnv.variables())) {
-            if (this.dataMasker.maybeSecretKey(k)) {
-              values.push(v as string);
-            }
+        const keys = await this.swaggerAnalyzer.getAllSecretKey();
```
Comment (raych1) [inline lib/testScenario/postmanCollectionRunnerClient.ts:483]: Why remove the blob option check?

File: ChangeLog.md (hunk 1 above)
Comment (ruowan) [reply ChangeLog.md:5]: good point.

File: lib/testScenario/postmanCollectionRunnerClient.ts (hunk 2 above)
Comment (ruowan) [reply lib/testScenario/postmanCollectionRunnerClient.ts:483]: not remove. Just move before uploadBlob

**Review by raych1 — APPROVED** [review]

**Pull request merged**